src/components/ButtonPanel.js

```diff
@@ -3,33 +3,33 @@ import Button from './Button';
 
 const ButtonPanel = () => (
   <div>
-    <div className="group1">
-      <Button name="AC" />
-      <Button name="+/-" />
-      <Button name="%" />
+    <div className="group">
+      <Button name="AC" color="#e0e0e0" />
+      <Button name="+/-" color="#e0e0e0" />
+      <Button name="%" color="#e0e0e0" />
       <Button name="÷" />
     </div>
-    <div className="group2">
-      <Button name="7" />
-      <Button name="8" />
-      <Button name="9" />
-      <Button name="X" />
+    <div className="group">
+      <Button name="7" color="#e0e0e0" />
+      <Button name="8" color="#e0e0e0" />
+      <Button name="9" color="#e0e0e0" />
+      <Button name="x" />
     </div>
-    <div className="group3">
-      <Button name="4" />
-      <Button name="5" />
-      <Button name="6" />
+    <div className="group">
+      <Button name="4" color="#e0e0e0" />
+      <Button name="5" color="#e0e0e0" />
+      <Button name="6" color="#e0e0e0" />
       <Button name="-" />
     </div>
-    <div className="group4">
-      <Button name="1" />
-      <Button name="2" />
-      <Button name="3" />
+    <div className="group">
+      <Button name="1" color="#e0e0e0" />
+      <Button name="2" color="#e0e0e0" />
+      <Button name="3" color="#e0e0e0" />
       <Button name="+" />
     </div>
-    <div className="group5">
-      <Button name="0" />
-      <Button name="." />
+    <div className="group">
+      <Button name="0" wide color="#e0e0e0" />
+      <Button name="." color="#e0e0e0" />
```

Comment:
When you set default props for a component you can avoid passing that prop when it is not needed that is if it should use the default value anyway

Reply:
In the requirements the default value is orange : color: '#f5913e', so there I'm passing the light gray color for the rest of the buttons.